src/iceberg_seals/search/predicting/predict_sealnet.py

```diff
@@ -274,8 +292,7 @@ def main():
                                                                 args.model_name)))
 
     # run validation to get confusion matrix
-    predict_patch(input_image=args.input_image, model=model,
-                  input_size=model_archs[args.model_architecture]['input_size'],
+    predict_patch(model=model, input_size=model_archs[args.model_architecture]['input_size'],
```

Comment:
`save_heatmaps` is not passed here, although it is given as an input. It should be added.